Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -544,10 +560,26 @@ class LoginDataSource: NSObject, UITableViewDataSource {
     }
 
     @objc func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = ThemedTableViewCell(style: .subtitle, reuseIdentifier: nil)
+        let cell = ThemedTableViewCell(style: .subtitle, reuseIdentifier: CellReuseIdentifier)
 
         if indexPath.section == LoginsSettingsSection {
-            indexPath.row == 0 ? boolSettings.0.onConfigureCell(cell) : boolSettings.1.onConfigureCell(cell)
+            let hideSettings = searchController?.isActive ?? false || tableView.isEditing
+            let setting = indexPath.row == 0 ? boolSettings.0 : boolSettings.1
+            setting.onConfigureCell(cell)
+            if hideSettings {
+                cell.isHidden = true
+            }
+
+            // Fade in the cell while dismissing the search or the cell showing suddenly looks janky
+            if isDismissSearchInProgress {
```

Comment:
The changes here are all cosmetic, related to smoothly re-displaying the hidden settings cells.
The hiding of the cells looks ok to me, but re-showing is ugly unless the cells are faded in.